client/src/game/star.js

```diff
@@ -368,8 +369,8 @@ class Star extends EventEmitter {
     if (!this.data.naturalResources) {
       return 0
     }
-
-    return Math.min(Math.floor(this.data.naturalResources / 45 * 3), 5) // Anything over 45 gets 3 planets
+    let averageNaturalResources = (this.data.naturalResources.economy + this.data.naturalResources.industry + this.data.naturalResources.science ) / 3;
```

Comment:
This would be nice in a function as its used above in another place.

Reply:
_getAverageNaturalResources()

Reply:
i was thinking about having rings of different colours, but then that would look strange since the players also have colours...

Reply:
Made the function _calculateNaturalResources
But its done